projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts

```diff
@@ -106,8 +107,12 @@ export class AddressBookComponent implements OnInit {
     );
   }
 
-  setAddressAsDefault(addressId: string): void {
-    this.service.setAddressAsDefault(addressId);
+  setAddressAsDefault(address: Address): void {
```

Comment:
the parameter has been changed, please add a note to migration doc

Reply:
Added